### PR TITLE
feat(store): add adorama

### PR DESCRIPTION
**Pull request by bmalaski**
### Description

Added adorama a store similar to bh. They have been adding and ordering cards slowly. 

### Testing
Tested the store, but only works with stealth mode from PR #83

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,11 +28,11 @@ The purpose of this bot is to get an Nvidia card. It tries multiple things to do
 
 > :point_right: You may get false positives from time to time, so I apologize for that. The library currently waits for all calls to be completed before parsing, but sometimes this can have unknown behavior. Patience is a virtue :)
 
-| | **Amazon** | **ASUS** | **EVGA** | **Best Buy** | **B&H** | **Micro Center** | **Newegg** | **Nvidia** |
-|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
-| **3070**| | | | | | | | |
-| **3080** | `✔` | `✔` | `✔` | `✔` | `✔` | `✔` | `✔` | `✔` |
-| **3090** | | | | | | | | |
+| | **Adorama** | **Amazon** | **ASUS** | **EVGA** | **Best Buy** | **B&H** | **Micro Center** | **Newegg** | **Nvidia** |
```
Comment (jef) [inline README.md:31]: Do you mind also adding to the list of stores below?

**Review by jef — CHANGES_REQUESTED** [review]
Just that one documentation update and looks good to me!

File: README.md [hunk 2]
```diff
@@ -88,10 +88,11 @@ Here is a list of variables that you can use to customize your newly copied `.en
 
 | **Stores** | **Environment variable** |
 |:---:|:---:|
-| Best Buy | `bestbuy`|
+| Adorama | `adorama`|
 | Amazon | `amazon`|
 | Amazon.ca | `amazon-ca`|
 | ASUS | `asus` |
+| Best Buy | `bestbuy`|
```
Comment (jef) [inline README.md:95]: :sweat_smile: Thanks!

File: README.md (hunk 1 above)
Comment (bmalaski) [reply README.md:31]: Didn't notice that, I added it to the stores list.

**Review by jef — APPROVED** [review]
Looks good here! Thanks :smile:

**Pull request merged**